Makefile

```diff
@@ -59,12 +59,12 @@ clean-all: clean-channel clean-common clean-controller clean-dispatcher
 
 
 #
-# Dep / Fmt / Vet / Lint
+# Mod / Fmt / Vet / Lint
 #
 
-dep:
+mod:
 	@echo 'Ensuring Dependencies'
-	cd $(BUILD_ROOT); dep ensure -v
+	cd $(BUILD_ROOT); go mod tidy; go mod vendor
```

Comment:
nit, but should this use the &&?
go mod tidy && go mod vendor?

Reply:
👍